src/SB/Core/x/xShadow.h

```diff
@@ -33,3 +33,11 @@ void xShadowManager_Reset();
 void xShadowManager_Render();
 
 #endif
+
+void xDrawSphere(const xVec3*, F32, U32)
```

Comment:
this function definition is causing your ci errors (run `ninja all_source` to test locally). I think it's supposed to be a WEAK function. check if the symbol has a `w` icon before it in objdiff.

Reply:
There might be other WEAK functions in this TU too so look for that.